pages/partials/mathjax.ejs

```diff
@@ -16,12 +24,32 @@ var outputComponent = 'output/chtml';
     outputComponent = 'output/svg';
 <% } %>
 
+// If the user overrides the renderer preference with the pop-up menu, reset
+// it when the page reloads. The menu choice is useful for investigating the
+// differences in rendering interactively, but later on the menu could be
+// deactivated for users entirely.
+try {
+    const settingsKey = 'MathJax-Menu-Settings';
+    const settings = JSON.parse(localStorage.getItem(settingsKey)) || {};
+    delete settings['renderer'];
+    const settingsJson = JSON.stringify(settings);
+    if (settingsJson === '{}') {
+        localStorage.removeItem(settingsKey);
+    } else {
+        localStorage.setItem(settingsKey, settingsJson);
+    }
+} catch(e) { }
```

Comment:
Going by what @trombonekenny said, we should allow users to change this if they must (maybe it allows them to circumvent *another* issue with rendering). I like his suggestion of adding a "Reset MathJax settings" button to the user settings page.

Reply:
I can look into putting the reset on the settings page instead of here. However, I should note that with this as-is, the page will load successfully even if the rendering looks bad (thanks to the local, non-global cache), and then the user can use the popup menu to change the renderer for the duration of the page view. It will reset if they reload the page though.

For the sake of merging this PR sooner, should we just remove this chunk here?

Reply:
Yeah, I'd err towards removing this. If the user _can_ fix rendering by changing the renderer, I'd rather they have the ability to persist that.

Reply:
Looks like @trombonekenny has covered this here: https://github.com/PrairieLearn/PrairieLearn/pull/7286